### ISSUE TITLE
buildPlugin for svg

### ISSUE THREAD

**Issue by gadicc** [PEM--/fview-svg#1]
Sorry, this isn't directly related to the project, but it occured to me during the README, re:
- removing transformations
- optimizing, removing editor bloat, etc.

We could make a build plugin for this, where the author drops in a .svg anywhere in the project and the build plugin optimizes, compiles and bundles with the other JS, as a template called say "svg_filename" or something.

Just wanted this written down somewhere :>

**Comment by PEM--** [comment]
Ho ho ho, that would be definitely an incredible addition. Designers would put their SVG directly in a folder and makes them available for animators to put them in motion.

**Comment by gadicc** [comment]
yeah, what i love about meteor is that I don't have to spend time on minification/optimization etc and my source repo has all original content - i.e. it's my backup too - and no build junk.  also, assuming those techniques are available via npm modules, this is super easy to do.  otherwise they just need the bin in their path.  not sure exactly what's involved with getting inkscape etc to normalize rotations and if we can do it externally.  anyway, we'll look into it :)
